### PR TITLE
Add reporting chart beta component

### PR DESCRIPTION
**Pull request by neerajsamtani-stripe**
adds ReportingChart to package
- https://docs.google.com/document/d/1ovoyOd3H6A32a1D5YSHv5dYUZDootpdWJRE1p9xhSpg/edit?usp=sharing

### CODE REVIEW

File: types/shared.d.ts [hunk 1]
```diff
@@ -473,4 +473,5 @@ export type ConnectElementTagName =
   | "balances"
   | "payouts-list"
   | "app-install"
-  | "app-viewport";
+  | "app-viewport"
```
Comment (jorgea-stripe) [inline types/shared.d.ts:476]: This PR is missing all of the setters related to this component

**Review by jorgea-stripe — CHANGES_REQUESTED** [review]

File: types/shared.d.ts (hunk 1 above)
Comment (neerajsamtani-stripe) [reply types/shared.d.ts:476]: Added the setters in `types/config.ts`

**Review by jorgea-stripe — APPROVED** [review]
LGTM, but please don't merge until the component is ready! I don't have another way of removing the "waiting for changes"

**Review by torrance-stripe — COMMENTED** [review]
this still LGTM to me!

**Pull request merged**